python/cudf/cudf/core/series.py

```diff
@@ -2254,16 +2258,14 @@ def count(self, level=None, **kwargs):
         >>> ser.count()
         5
         """
-
         if level is not None:
             raise NotImplementedError("level parameter is not implemented yet")
 
         return self.valid_count
 
     @_cudf_nvtx_annotate
     def mode(self, dropna=True):
-        """
-        Return the mode(s) of the dataset.
+        """Return the mode or modes of the dataset.
```

Comment:
Pandas says "Return the mode(s) of the Series." Does this conflict with a pydocstyle rule?

We're also missing a line in the description compared to pandas' API: "The mode is the value that appears most often. There can be multiple modes."

https://pandas.pydata.org/docs/reference/api/pandas.Series.mode.html

Reply:
It conflicts with a rule in a really dumb way. Any time that there are parentheses in the first line of the docstring pydocstyle interprets that as the line containing a function signature. It's an overly broad way of catching signatures (see PyCQA/pydocstyle#284). I figured that just rewording was a better solution than adding a noqa, but I could add the noqa instead if you prefer. What do you think?

I did not notice the extra lines and can add that once we decide how to resolve the plural issue.